server.js

```diff
@@ -18,9 +18,71 @@ app.use(express.json());
 
 // Start defining your routes here
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+  res.send("Happy thoughts on repeat! Type '/thoughts' to see them!");
 });
 
+const ThoughtsSchema = new mongoose.Schema({
+  message: {
+    type: String,
+    required: true,
+    minlenght: 4,
+    maxlength: 140,
+    trim: true
+  },
+  hearts: {
+    type: Number,
+    default: 0
+  },
+  createdAt: {
+    type: Date,
+    default: () => new Date()
+  }
+})
+
+const Thoughts = mongoose.model("Thoughts", ThoughtsSchema)
+
+app.post("/thoughts", async (req, res) => {
+  const { message } = req.body
+  console.log(req.body)
+  try {
+    const newThought = await new Thoughts({ message: message }).save()
+    res.status(201).json({ success: true, response: newThought })
+  } catch (error) {
+    res.status(400).json({ success: false, response: error })
+  }
+})
+
+app.get("/thoughts", async (req, res) => {
+  const response = {
+    success: true,
+    body: {}
+  }
+  try {
+    response.body = await Thoughts.find().sort({ createdAt: "desc" }).limit(20).exec();
+    res.status(200).json({
+      success: true,
+      body: response
```

Comment:
Why did you choose to put the "response" variable inside the body? When I look at your json file, it's a double success & body nested inside.